export/exportconfig.go

```diff
@@ -3,21 +3,23 @@ package export
 import (
 	cc "github.com/pivotalservices/cf-mgmt/cloudcontroller"
 	"github.com/pivotalservices/cf-mgmt/config"
-	"github.com/pivotalservices/cf-mgmt/organization"
-	"github.com/pivotalservices/cf-mgmt/space"
+	kOrg "github.com/pivotalservices/cf-mgmt/organization/constants"
+	kSpace "github.com/pivotalservices/cf-mgmt/space/constants"
 	"github.com/pivotalservices/cf-mgmt/uaac"
+	"github.com/pivotalservices/cf-mgmt/utils"
 	"github.com/xchapter7x/lo"
 )
 
 //NewExportManager Creates a new instance of the ImportConfig manager
 func NewExportManager(
 	configDir string,
 	uaacMgr uaac.Manager,
-	cloudController cc.Manager) Manager {
+	cloudController cc.Manager, utilsMgr utils.Manager) Manager {
```

Comment:
This seems to be getting unwieldy.  As the consumer of a package, it should be easy for me to look at the public API and figure out how to use it.  The more interfaces we have, the harder my job is because I have to figure out how to provide implementations for those interfaces.